draft-schwartz-ds-glue.md

```diff
@@ -52,25 +52,30 @@ Existing nameserver deployments assume that the delegation response includes onl
 
 # Proposal
 
-This draft proposes a way to convey glue RRs in DS records, enabling authenticated delivery of arbitrary RR types as part of the delegation response.
+This draft proposes a way to convey glue RRSets in DS records, enabling authenticated delivery of arbitrary RR types as part of the delegation response.
```

Comment:
Either in this sentence (or the next sentence) clarify if all child zone RRs to be encoded end up in a single DSGLUE RR or multiple DSGLUE RRs.

Reading line 59 below, it seems like each child zone RR ends up in its own DSGLUE RR.